Geometry/HGCalCommonData/plugins/DDHGCalTBModule.h

```diff
@@ -0,0 +1,60 @@
+#ifndef HGCalCommonData_DDHGCalTBModule_h
+#define HGCalCommonData_DDHGCalTBModule_h
+
+#include <map>
+#include <string>
+#include <vector>
+#include "DetectorDescription/Base/interface/DDTypes.h"
+#include "DetectorDescription/Algorithm/interface/DDAlgorithm.h"
+#include <unordered_set>
+
+class DDHGCalTBModule : public DDAlgorithm {
+ 
+public:
+  //Constructor and Destructor
+  DDHGCalTBModule(); //const std::string & name);
+  virtual ~DDHGCalTBModule();
+  
+  void initialize(const DDNumericArguments & nArgs,
+                  const DDVectorArguments & vArgs,
+                  const DDMapArguments & mArgs,
+                  const DDStringArguments & sArgs,
+                  const DDStringVectorArguments & vsArgs);
+  void execute(DDCompactView& cpv);
+
+protected:
+
+  void          constructLayers (DDLogicalPart, DDCompactView& cpv);
+  double        rMax(double z);
+  void          positionSensitive(DDLogicalPart& glog, int type, double rin,
+				  double rout, DDCompactView& cpv);
+
+private:
+
+  std::vector<std::string> wafer;         //Wafers
```

Comment:
hi @bsunanda - please follow a naming convention for class member variables (eg, in CMS its often blah_)
Thanks